user/invite-new.md

```diff
@@ -4,6 +4,32 @@ description: How to invite a new user
 
 # Invite user
 
-Explanation
+You can invite new user to your company:
```

Comment:
This should read `You can invite a new user to your company`.

Reply:
Fixed, thanks